src/taco-livereload/lib/utils/homePage/bower_components/q/.npmignore

```diff
@@ -0,0 +1,23 @@
+node_modules
```

Comment:
We do not need 'bower_components', it was just a way to acquire the dependencies.